src/absil/ilwritepdb.fsi

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+/// The ILPdbWriter 
+module internal Microsoft.FSharp.Compiler.AbstractIL.ILPdbWriter 
+
+open Microsoft.FSharp.Compiler.AbstractIL.IL 
+open Microsoft.FSharp.Compiler.ErrorLogger
+open Microsoft.FSharp.Compiler.Range
+open System.Collections.Generic 
+
+type PdbDocumentData = ILSourceDocument
+
+type PdbLocalVar = 
+    { Name: string
+      Signature: byte[] 
+      /// the local index the name corresponds to
+      Index: int32  }
+
+type PdbMethodScope = 
+    { Children: PdbMethodScope array
+      StartOffset: int
+      EndOffset: int
+      Locals: PdbLocalVar array
+      (* REVIEW open_namespaces: pdb_namespace array *) }
```

Comment:
Can you look into something for me?  This reminds me that the PDB format has an entry that F# has never filled in for the "open namespaces" within a method scope.  This records the active "using" statements for a piece of code.  I believe this entry is used to help interpret the stuff entered into the "Watch" and "Immediate" windows of the VS debugger.  So if in C# you have "using System" then you can put "Console.In" in the watch window

In F# I think this doesn't work - but only because we never fill in the "open namespaces" entry in the PDB format and write them out.  I assume there is a matching "TODO" in the portable PDB and MDB debug formats.  It would be good to plan ahead at this point and make sure the basic writers can emit this information, and then plumb it through fromm the type checker at a later point.

The "open namespaces" information wouldn't be too hard to propagate down.  But we've just never done it.  

Thanks